sick-fits/backend/.nvmrc

```diff
@@ -0,0 +1 @@
+v16.18.1
```

Comment:
My suggestion would be to make this just "v16", so that newer versions of version 16 can automatically be picked up without needing to come back and change this.

Reply:
Thanks for the review, that makes sense. I've updated the PR with your suggestion